Cpix/ContentKeyCollection.cs

```diff
@@ -24,63 +24,68 @@ protected override XmlElement SerializeEntity(XmlDocument document, XmlNamespace
 				KeyId = entity.Id,
 				ExplicitIv = entity.ExplicitIv,
 				CommonEncryptionScheme = entity.CommonEncryptionScheme,
-				Data = new DataElement
-				{
-					Secret = new SecretDataElement()
-				}
 			};
 
-			if (Document.Recipients.Any())
+			if (entity.Value != null)
```

Comment:
Perhaps leave a comment here:
"We support content keys without a value because many packagers create such documents for requesting keys from key services, which then fill in the value."